src/main/java/com/nirima/jenkins/plugins/docker/DockerContainerWatchdog.java

```diff
@@ -121,7 +121,7 @@ protected void execute(TaskListener listener) throws IOException, InterruptedExc
             return;
         }
 
-        LOGGER.info("Docker Container Watchdog has been triggered");
+        LOGGER.debug("Docker Container Watchdog has been triggered");
```

Comment:
For some reason this plugin uses SLF4J, unlike the rest of the Jenkins ecosystem which uses JUL.